user-guide/Cloud_Platform/CloudAdminApp/Managing_DCP_keys.md

```diff
@@ -2,34 +2,59 @@
 uid: Managing_DCP_keys
 ---
 
-# Managing dataminer.services keys
+# dataminer.services keys
```

Comment:
I'd prefer to keep the original title here, as the page includes everything from viewing the keys to managing them, and we often put such things under one "managing" header. Otherwise, we'll have to change this title in the TOC as well, and it will stand out like a sore thumb between all the other titles in that section:
![image](https://github.com/SkylineCommunications/dataminer-docs/assets/94605575/482cbbe9-9426-41a9-bd30-c763105efd0e)

Reply:
done